addons/website_theme_install/static/src/less/website_theme_install.less

```diff
@@ -1,42 +1,173 @@
+@o-theme-kanban-gray: #fcfcfc;
+
 .o_kanban_view.o_theme_kanban {
+    background-color: @o-theme-kanban-gray;
+
     &.o_kanban_ungrouped {
         .o-justify-content(center);
 
         .o_kanban_record {
-            .o-flex(0, 0, auto);
-            width: 100%;
-            @media (min-width: @screen-sm-min) {
-                width: 45%;
+            .o-flex(0, 1, auto);
+            width: 250px;
+
+            &:first-child, &:nth-child(2) {
+                @media (max-width: @screen-xs-max) {
+                    margin-top: 10px;
+                }
             }
         }
     }
 
     .o_kanban_record {
-        padding: 0;
+        margin: 1% 2% 2%;
+        box-shadow: none;
+        background: none;
+        border: none;
+
+        .o_browser_bar:before {
+            content: "•••";
+            display: block;
+            padding-left: 10px;
+            color: darken(@o-theme-kanban-gray, 10%);
+            font: 12px/20px Courier, monospace;
+            letter-spacing: -1px;
```

Comment:
mmm yes!
First of all `letter-spacing` it's not "ugly" :P 
In general it's perfect to increase readability when the font is used for purposes different from the intended one (eg. a font made specifically for huge ADV headings that is used for tiny `h6` titles too).
At one point  you will have to decide if import another font or just apply a readability css fix for specific case scenarios...

In this particular case: we are not talking about real "text". The 3 dots are more a graphical element... so those guys definitely belong to my jurisdiction ^^ !